addons/website/views/website_templates.xml

```diff
@@ -1864,7 +1864,7 @@
 <!-- Scroll to Top option -->
 <template id="option_footer_scrolltop" inherit_id="website.layout" active="False">
     <xpath expr="//div[@id='footer']" position="after">
-        <div id="o_footer_scrolltop_wrapper" class="container h-0 d-flex align-items-center justify-content-center">
+        <div id="o_footer_scrolltop_wrapper" class="container d-flex align-items-center justify-content-center">
```

Comment:
I don't think the commit message explanation is right, neither is the fix:

- I don't think BS4 had a h-0 class, we just extended BS4 ones to have it
- You should check why that extension was removed and probably restore it instead

Reply:
Got curious: [it's this code](https://github.com/odoo/odoo/commit/fe4f1efb70c0a9c4ebcb2b51fe090c4731c399b4#diff-23677f06a9aaf2e3395e957cfb9580ce6fcc9aba2635b560fe8f15fa27707313R17) that has not been adapted to extend BS5 utilities the right way (cc @adr-odoo just so you know)

Reply:
I removed this commit from this branch, and moved it here: https://github.com/odoo/odoo/pull/101541